templates/user/settings/profile.tmpl

```diff
@@ -47,27 +47,54 @@
 					<input id="location" name="location"  value="{{.SignedUser.Location}}">
 				</div>
 
-					<div class="field">
-						<label for="language">{{.i18n.Tr "settings.language"}}</label>
-						<div class="ui language selection dropdown" id="language">
-							<input name="language" type="hidden" value="{{.SignedUser.Language}}">
-							{{svg "octicon-triangle-down" 14 "dropdown icon"}}
-							<div class="text">{{range .AllLangs}}{{if eq $.SignedUser.Language .Lang}}{{.Name}}{{end}}{{end}}</div>
-							<div class="menu">
-							{{range .AllLangs}}
-								<div class="item{{if eq $.SignedUser.Language .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
-							{{end}}
-							</div>
+				<div class="field">
+					<label for="language">{{.i18n.Tr "settings.language"}}</label>
+					<div class="ui language selection dropdown" id="language">
+						<input name="language" type="hidden" value="{{.SignedUser.Language}}">
+						{{svg "octicon-triangle-down" 14 "dropdown icon"}}
+						<div class="text">{{range .AllLangs}}{{if eq $.SignedUser.Language .Lang}}{{.Name}}{{end}}{{end}}</div>
+						<div class="menu">
+						{{range .AllLangs}}
+							<div class="item{{if eq $.SignedUser.Language .Lang}} active selected{{end}}" data-value="{{.Lang}}">{{.Name}}</div>
+						{{end}}
 						</div>
 					</div>
+				</div>
+
+				<div class="ui divider"></div>
+				<!-- private block -->
+
+				<div class="field">
+					<label for="security-private"><strong>{{.i18n.Tr "settings.privacy"}}</strong></label>
+				</div>
+
+				<div class="inline field {{if .Err_Visibility}}error{{end}}">
+					<span class="inline required field"><label for="visibility">{{.i18n.Tr "settings.visibility"}}</label></span>
+					<div class="ui selection type dropdown">
+						<input type="hidden" id="visibility" name="visibility" value="{{.visibility}}">
```

Comment:
`value="{{.visibility}}"` always renders to `value=""` for me. Not sure why it works in the admin panel but not here, as I don't understand where the `.visibility` var should be set

Reply:
Yes, it's should be something like `.SignedUser.Visibility`